File: C#-OOP-Basics/Problems Course Instance 2018/DefiningClasses/RawData/Tire.cs
namespace RawData
{
    class Tire
    {
        private double pressure;
        private int age;

        public Tire() { }

        public Tire(int age, double pressure)
        {
            this.Age = age;
            this.Pressure = pressure;
        }

        public int Age
        {
            get { return age; }
            set { age = value; }
        }

        public double Pressure
        {
            get { return pressure; }
            set { pressure = value; }
        }
    }
}

File: Java-OOP-Basics/Problems Course Instance 2016_2017/src/Lesson01DefiningClasses/Exercise/pr06_raw_data/model/Tire.java
package Lesson01DefiningClasses.Exercise.pr06_raw_data.model;

public class Tire {
    private double pressure;
    private int age;

    Tire(double pressure, int age) {
        this.pressure = pressure;
        this.age = age;
    }

    public double getPressure() {
        return pressure;
    }
}
